controlplane/package.json

```diff
@@ -14,11 +14,13 @@
     "debug": "tsx watch src/index.ts",
     "build": "del dist && tsc",
     "seed": "API_KEY=cosmo_669b576aaadc10ee1ae81d9193425705 tsx src/bin/seed.ts",
+    "seed:billing": "tsx src/bin/seed-billing.ts",
```

Comment:
Please check if the seed is also executed on the full-demo and helm chart.

Reply:
I don't think this should be added to the demo and just treat the demo as an on-prem deployment without billing.

Not sure about the helm chart though, the seed script should be run one time when we deploy our cloud.
I recommend you set this up.